src/silo/prepare_dataset.cpp

```diff
@@ -398,6 +411,18 @@ void silo::sortChunks(
          output_filename += silo::buildChunkString(chunk.part, chunk.chunk) + ZSTDFASTA_EXTENSION;
          sequence_outputs.emplace_back(output_filename, reference_sequence);
       }
+      for (const auto& [aa_name, reference_sequence] : reference_genomes.aa_sequences) {
+         std::filesystem::path input_filename = input_folder;
+         input_filename += "gene_" + aa_name + std::filesystem::path::preferred_separator;
+         input_filename += silo::buildChunkString(chunk.part, chunk.chunk) + ZSTDFASTA_EXTENSION;
+         sequence_inputs.emplace_back(input_filename, reference_sequence);
+
+         std::filesystem::path output_filename = output_folder;
+         output_filename += "gene_" + aa_name + std::filesystem::path::preferred_separator;
+         create_directory(output_filename);
+         output_filename += silo::buildChunkString(chunk.part, chunk.chunk) + ZSTDFASTA_EXTENSION;
+         sequence_outputs.emplace_back(output_filename, reference_sequence);
+      }
```

Comment:
Same here

Reply:
Must of the code is managing the filenames and creating the directory. I will tackle that this is at most two lines of code in the body in #156, which should make it idiomatic enough?